azure-service-bus-management/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,61 @@
+= Azure Service Bus Management Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft Azure Service Bus Management (Azure Service Bus Management Connector) enables management integration with Azure Service Bus on the cloud. This connector connects to the Azure REST API and supports management operations for queues, topics, and subscriptions. In addition, you can use the built-in management API to discover Azure Service Bus objects automatically and provision them.
+
+For compatibility information, see the Azure Service Bus Management Connector Release Notes.
```

Comment:
Kim told me to leave the line of text and move the link to the See Also. This is not high priority right now, and I'll need to update our template.